README.md

```diff
@@ -184,6 +186,49 @@ repositories {
 
 The builds are distributed using JCenter through Bintray [JDA JCenter Bintray](https://bintray.com/dv8fromtheworld/maven/JDA/)
 
+### Development Builds
+
+The builds on the `development` branch can be found on [jitpack](https://jitpack.io/#DV8FromTheWorld/JDA)
+
+**Maven**
+```xml
+<dependency>
+    <groupId>com.github.DV8FromTheWorld</groupId>
+    <artifactId>JDA</artifactId>
+    <version>COMMIT</version>
+</dependency>
+```
+```xml
+<repository>
+    <id>jcenter</id>
```

Comment:
These builds will only be on jitpack, not on jcenter.

Reply:
jcenter is added for transitive dependencies which are not resolvable from jitpack

Reply:
right...